nomad/structs/structs.go

```diff
@@ -3336,6 +3347,10 @@ type TaskGroup struct {
 	// Affinities can be specified at the task group level to express
 	// scheduling preferences.
 	Affinities []*Affinity
+
+	// Spread can be specified at the task group level to express spreading
```

Comment:
Will it be possible to specify on job level and have the same behavior as `update{}` where it cascade / inherit down from `job -> spread` to `job -> group -> spread`? 

Similar question for `affinity{}`

Reply:
@jippi yes this and affinities both work similar to the update stanza and cascade down.